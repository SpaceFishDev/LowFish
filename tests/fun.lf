i32 main(i32 argc, ptr argv)
{
}

