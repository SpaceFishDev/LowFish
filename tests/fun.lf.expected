<NODE( 'PROGRAM', PROGRAM )>
{
    <NODE( 'FUNCTION', FUNCTION )>
    {
        <NODE( 'i32', TYPE )>
        <NODE( 'main', TOKEN_NODE )>
        <NODE( 'VARDECL', VARDECL )>
        {
            <NODE( 'i32', TYPE )>
            <NODE( 'argc', TOKEN_NODE )>
        }
        <NODE( 'VARDECL', VARDECL )>
        {
            <NODE( 'ptr', TYPE )>
            <NODE( 'argv', TOKEN_NODE )>
        }
        <NODE( 'BLOCK', BLOCK )>
    }
}
